recipe/windows_cmake_install_bindir.patch

```diff
@@ -0,0 +1,37 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 3fd9d70..f8bf221 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -399,6 +399,7 @@ endif ()
+ if (OS_WINDOWS)
+   set (RUNTIME_INSTALL_DIR Bin)
+   set (LIBRARY_INSTALL_DIR Lib)
```

Comment:
Sorry for the misunderstanding I thought the comment was about `RUNTIME_INSTALL_DIR`. The patch is OK the way it is. It is and should be:
```
set (RUNTIME_INSTALL_DIR Bin)
set (LIBRARY_INSTALL_DIR Lib)
set (ARCHIVE_INSTALL_DIR Lib)
```